### DISCUSSION TITLE
Survey - what SAAS apps need machine learning the most?

### DISCUSSION THREAD

**Discussion by C0staTin** [mindsdb/mindsdb#2550] [Announcements]
Hey Everyone! :loudspeaker:
Please let us know what SAAS applications you would like MindsDB to work with, out of the box!
Please complete the 10 min survey - https://surveyking.com/w/463myf9
As a sign of our appreciation, you will be entered into a prize draw to win an Amazon $500 gift :gift: voucher.
Moreover, 20 people with the most detailed responses will get these nice MindsDB t-shirts :shirt:.
:page_with_curl: [Take the Survey now](https://surveyking.com/w/463myf9)!

P.S. The survey will be open until July 31st, end of the day. 
We will make the draw on the first week of August and communicate the winner in this thread.

**Comment by prathikshetty2002** [comment]
Interesting !

**Comment by C0staTin** [comment]
Hi Everyone! :loudspeaker:
Thanks to those who responded to the SaaS survey :bookmark_tabs: 
Now we are happy to announce the results of the prize draw :gift:!
And the winner is... :hourglass_flowing_sand: Muhammad Misbahuddin! :bangbang:
Congratulations!!! :tada::confetti_ball::champagne:
You will receive a $500 Amazon gift voucher. :e-mail:

We have carefully reviewed responses and have 20 cool MindsDB T-Shirts :shirt: ready to go to people with the most detailed & valuable answers
![Amazon 500 Gift card](https://user-images.githubusercontent.com/12409467/182737621-44e3b040-7ad8-46b9-bac9-ccac7861e6d1.jpg)
![Tshirt_mindsdb-both](https://user-images.githubusercontent.com/12409467/182737636-3ffeb0c9-0295-4980-b27f-02a40a8ed7a8.jpg)
!

**Comment by prathikshetty2002** [comment]
Thanks , really enjoyed this survey! waiting for more such contributions to the community <3 🤩
